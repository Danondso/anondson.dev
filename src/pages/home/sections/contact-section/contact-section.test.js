import React from 'react';
import renderer from 'react-test-renderer';
import ContactSection from './contact-section';

describe('Contact Section Test Suite', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ContactSection />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
