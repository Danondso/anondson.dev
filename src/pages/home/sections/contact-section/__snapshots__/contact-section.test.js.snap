// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Section Test Suite renders correctly 1`] = `
<section
  alt="Contact Section of anondson.dev"
  style={
    Object {
      "backgroundColor": "var(--background-green)",
      "boxShadow": "0 3px 2px rgba(0, 0, 0, 0.15)",
      "padding": "4rem",
    }
  }
>
  <h1>
    Let's get in touch.
  </h1>
  <section
    style={
      Object {
        "fontSize": "1.05rem",
        "maxWidth": "24rem",
        "padding": "1rem",
      }
    }
  >
    Drop me a line
     
    <a
      className="link"
      href="mailto:dublin@anondson@gmail.com"
      rel="noopener noreferrer"
      target="_blank"
    >
       here. 
    </a>
    If you want to get really professional about it here's my
    <a
      className="link"
      href="https://www.linkedin.com/in/dublin-anondson-4b562793/"
      rel="noopener noreferrer"
      target="_blank"
    >
       LinkedIn.
    </a>
  </section>
</section>
`;
